draft-bbs-signatures.md

```diff
@@ -686,9 +683,9 @@ In addition, ciphersuites MUST specify unique domain separation tags for hash\_t
 
 BBS signatures can be implemented on any pairing-friendly curve. However care MUST be taken when selecting one that is appropriate, this specification defines a ciphersuite for using the BLS12-381 curve in (#ciphersuites) which as a curve achieves around 117 bits of security according to a recent NCC ZCash cryptography review [@ZCASH-REVIEW].
 
-## Security of spkGen
+## Security of Proofs generated by ProofGen
 
-The BBS proof, as returned by spkGen, is a zero-knowledge proof-of-knowledge [@CDL16]. This guarantees that no information will be revealed about the signature itself or the undisclosed messages, from the output of spkGen. Note that the security proofs in [@CDL16] work on type 3 pairing setting. This means that G1 should be different from G2 and with no efficient isomorphism between them.
+The proof, as returned by ProofGen, is a zero-knowledge proof-of-knowledge [@CDL16]. This guarantees that no information will be revealed about the signature itself or the undisclosed messages, from the output of ProofGen. Note that the security proofs in [@CDL16] work on type 3 pairing setting. This means that G1 should be different from G2 and with no efficient isomorphism between them.
```

Comment:
Minor nit: Many crypto papers use unrevealed instead of undisclosed. Should align with the literature?

Reply:
"Selective disclosure" is a common expression for this feature, which naturally leads to "undisclosed" messages; I'd prefer keeping this as is.

Reply:
I agree with @christianpaquin in most contexts I've seen the term selective disclosure is now quite well understood I'd prefer we align to this consensus